database_cleaner.gemspec

```diff
@@ -51,8 +51,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency "couch_potato"
   s.add_development_dependency "sequel", "~> 3.21.0"
   s.add_development_dependency 'ohm', '~> 0.1.3'
-  s.add_development_dependency 'guard-rspec'
```

Comment:
@botandrose Let's keep `guard-rspec` for now.

Reply:
Oh, yeah, I remember what happened now. Bumping `rspec` required bumping `guard-rspec` which required bumping `guard` which requires Ruby 2.2+. Ugh. I'm going to see if there's some version in the middle that will work.